apps/web/pages/success.tsx

```diff
@@ -249,337 +249,330 @@ export default function Success(props: SuccessProps) {
     return t("emailed_you_and_attendees" + titleSuffix);
   }
   const userIsOwner = !!(session?.user?.id && eventType.users.find((user) => (user.id = session.user.id)));
-  const { isReady, Theme } = useTheme(isSuccessBookingPage ? props.profile.theme : "light");
+  const {} = useTheme(isSuccessBookingPage ? props.profile.theme : "light");
```

Comment:
```suggestion
  useTheme(isSuccessBookingPage ? props.profile.theme : "light");
```

If we're not destructuring anything we can just do this